### PR TITLE
Support for some SoHL-based mods and more

### PR DESCRIPTION
**Pull request by hobokenn**

### CODE REVIEW

File: BunnymodXT/modules/ServerDLL.cpp [hunk 1]
```diff
@@ -1388,15 +1364,21 @@ void ServerDLL::DoMultiManagerAutoStop(const char *targetname)
...
 		|| (!std::strcmp(targetname, "exit_seq") && !std::strcmp(gameDir, "timeline2")) // Timeline 2
 		|| (!std::strcmp(targetname, "spawn_garg_sci_mm") && !std::strcmp(gameDir, "SteamLink")) // Uplink
-		|| (!std::strcmp(targetname, "medicosprey") && !std::strcmp(gameDir, "visitors"))) { // Visitors
-		if (CVars::bxt_timer_autostop.GetBool())
-			CustomHud::SetCountingTime(false);
-		Interprocess::WriteGameEnd(CustomHud::GetTime());
-		CustomHud::SaveTimeToDemo();
-		RuntimeData::Add(RuntimeData::GameEndMarker{});
+		|| (!std::strcmp(targetname, "medicosprey") && !std::strcmp(gameDir, "visitors")) // Visitors
+		|| (!std::strcmp(targetname, "fc_mm1") && !std::strcmp(gameDir, "hc"))) { // Hazardous Course 2
```
Comment (YaLTeR) [inline BunnymodXT/modules/ServerDLL.cpp:1508]: Seems like this should be in a different commit?

Comment (hobokenn) [reply BunnymodXT/modules/ServerDLL.cpp:1508]: it should be, but i just left it there, staging different parts took way too much time

**Comment by YaLTeR** [comment]
Thanks!
